### ISSUE TITLE
Taxonomic keys as independent deposits 

### ISSUE THREAD

**Issue by mguidoti** [plazi/arcadia-project#83]
I was talking to @myrmoteras and this might be a necessary addition: **to add taxonomic keys as separate, independent deposits on Zenodo, issuing a DOI.** 

**Motivation**: keys are in fact cited by treatments inside the same paper, just like a figure or table, they are not part of a given treatment - I think @tcatapano will agree with me on this. Additionally, they are cited by different papers that use that particular key to identify their studied material or even when they present an updated version of it (extremely common in taxonomy).

Thus, if you guys accept this idea, keys would be in Zenodo with their own DOIs, and they would be cited by the treatment and article pages just like figures. 

Thoughts?

@gsautter

**Comment by gsautter** [comment]
Considering the fact that we already have keys as separate treatments in TB, this shouldn't be all too hard to do ... my comment this morning was about treating the outcomes of the key leads (i.e., the taxa the key ends in) as treatment citations and link them accordingly.

**Comment by mguidoti** [comment]
I don't think we should treat cited taxa in key leads as treatment citations because the key entry shouldn't be considered a treatment in the first place. I see taxonomic keys, and consequently, key leads, just like figures: a source of information about that particular taxon concept but not a treatment on itself. 

So, I would vote for adding taxonomic key as independent depositions in Zenodo, give them a specific DOI, and cite them in the taxonomic treatments - **while also making them available for other people's work to cite.** 

In a way, that's liberating another very important part of taxonomic data.

**Comment by gsautter** [comment]
Treatment citations at key lead exist or not (I was more about linking), we _already_ have keys as treatments (if a special type of them) in TB, and as every treatment will go on Zenodo, so will the keys.

**Comment by gsautter** [comment]
Example for a standalone key that we handle a treatment in itself https://sandbox.zenodo.org/record/367787 (http://treatment.plazi.org/id/039A5C527B330730FC89FD439DB1F9F8)

**Comment by mguidoti** [comment]
Ok, just to wrap this up: Guido and I discussed this in person and yes, my idea is already implemented, which is really, really cool.

There are some somehow rare cases where the key is cited only by one treatment in the given paper where he treated the key as a part of the only treatment that cites it and not as a independent deposit - here my suggestion still stands, which is to give a DOI to these keys as well, but it's on the wish list and input from @tcatapano and @myrmoteras are necessary/welcome, _so perhaps we should continue to discuss this odd cases?_

For Guido's original thoughts on linking treatments to the key on Zenodo: he clarified with Alex and it'll be possible - so, awesome addition as well. 

Thanks, @gsautter, and correct me here if I've said anything wrong.

**Comment by myrmoteras** [comment]
We need to discuss first, come to a decision, and then implement such far reaching issues.

Check out new species<<a%20href='http://tb.plazi.org/GgServer/static/newToday.html'%20target='_blank'>http://tb.plazi.org/GgServer/static/newToday.html</a>> on Plazi<<a%20href='http://plazi.org'%20target='_blank'>http://plazi.org</a>>

________________________________
